Annex60/Fluid/Sensors/TemperatureTwoPort.mo

```diff
@@ -63,8 +64,9 @@ equation
   end if;
   // Output signal of sensor
   if dynamic then
-    if enableThermalLosses then
-      der(T) = (TMed-T)*k*tauInv + (TAmb-T)*tauLossInv;
+    if transferHeat then
+      // Note that Dymola will remove the term (TAmb-T)*tauHeaTraInv if tauHeaTraInv=0.
```

Comment:
That's odd since Dymola provides the option to change the parameter value after compilation and if you then change tauLossInv to be not equal to zero, the equation would be wrong?